doc/tutorials/gapi/oak_devices/oak_devices.markdown

```diff
@@ -0,0 +1,38 @@
+Using DepthAI Hardware /OAK depth sensors {#tutorial_gapi_oak_devices}
+=======================================================================================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_gapi_face_beautification}
+
+![hardwares](pics/oak.jpg)
+
+Depth sensors compatible with Luxonis DepthAI library are supported through OpenCV Graph API (or G-API) module. Depth map, RGB image and some other formats of output can be retrieved by using familiar interface of G-API module.
+
+In order to use DepthAI sensor with OpenCV you should do the following preliminary steps:
+-#  Install Intel DepthAI library/depthai-core (from here <https://github.com/luxonis/depthai-core>).
+
+-#  Configure OpenCV with DepthAI library support by setting WITH_OAK flag in CMake. If DepthAI library is found in install folders OpenCV will be built with depthai-core  (see a status WITH_OAK in CMake log).
+
+-#  Build OpenCV.
+
+Below an example to capture a frame and save in .png format:
+
+@code{.cpp}
+    cv::GFrame in;
+    cv::GFrame copy = cv::gapi::oak::copy(in);
+    cv::GMat out = cv::gapi::streaming::Y(copy);
+
+    auto args =  cv::compile_args(cv::gapi::oak::ColorCameraParams{},cv::gapi::oak::kernels());
+
+    auto pipeline = cv::Gcomputation(cv::Gin(in), cv::GOut(out)).compileStreaming(std::move(args));
```

Comment:
This code won't compile at least due to ``Gin` typo.
Also, it is worth mentioning what kind of headers required to run an OAK pipeline when OpenCV is compiled with OAK support.

Reply:
`Gcomputation` -> `GComputation` typo